homeassistant/components/androidtv/config_flow.py

```diff
@@ -224,13 +225,17 @@ def _async_init_form(self) -> FlowResult:
         """Return initial configuration form."""
 
         apps_list = {k: f"{v} ({k})" if v else k for k, v in self._apps.items()}
-        apps = {APPS_NEW_ID: "Add new", **apps_list}
+        apps = [{"label": "Add new", "value": APPS_NEW_ID}] + [
+            {"label": v, "value": k} for k, v in apps_list.items()
+        ]
         rules = [RULES_NEW_ID] + list(self._state_det_rules)
         options = self.options
 
         data_schema = vol.Schema(
             {
-                vol.Optional(CONF_APPS): vol.In(apps),
+                vol.Optional(CONF_APPS): selector(
+                    {"select": {"options": apps, "mode": "dropdown"}}
+                ),
```

Comment:
```suggestion
                vol.Optional(CONF_APPS): selector.SelectSelector(
                    selector.SelectSelectorConfig(options=apps, mode=selector.SelectSelectorMode.DROPDOWN),
```
Same with the one below and you could/should perhaps change them all to selectors now if you are anyway changing these two.

Reply:
More a suggestion to be consistent

Reply:
Ok, I'll do on next PR. Thanks.